### PR TITLE
 Warning upon listing below floor

### PR DESCRIPTION
**Pull request by magiclars-off**
Warns the user upon creating a listing more than 10% below floorprice.

image

Also fixes a small bug where parseFloat was fed a NaN after clearing the input 

image

@wyze @jcheese1

### CODE REVIEW

File: src/pages/inventory/[[...section]].tsx [hunk 1]
```diff
@@ -516,6 +545,9 @@ const Drawer = ({
                               }
                             })}
                           </div>
+                          {floorDifference > 10 && (
+                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floorpice`}</p>
```
Comment (wyze) [inline src/pages/inventory/[[...section]].tsx:549]:
```suggestion
                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floor price`}</p>
```
Typo here, and added a space between the words.

**Review by wyze — CHANGES_REQUESTED** [review]
Can we also maybe change the background color or the button for a little more visual indication that they should pay attention? Maybe there is a good shade of yellow that wouldn't look terrible. Since our theme is red.

**Comment by magiclars-off** [comment]
Thank you for the feedback, you are right. The warning didn't fit well in the UI

I made some changes

- moved the warning under the price input
- moved the royalties / share component down above the button and increased the font-size a little bit for readability

image

**Comment by wyze** [comment]
@magiclars-off Looks good, except can we clear the error if I remove the value from the input field?

**Comment by magiclars-off** [comment]
@wyze done

**Review by wyze — APPROVED** [review]
Awesome, thank you!

**Pull request merged**